src/utils/navigation.tsx

```diff
@@ -1,13 +1,40 @@
 import analytics from '@react-native-firebase/analytics';
 import {useNavigation} from '@react-navigation/native';
 import {useCallback} from 'react';
+import {
+  usePreferenceItems,
+  Preference_ScreenAlternatives,
+} from '../preferences';
+import {TabNavigatorParams} from '../navigation/TabNavigator';
+import {useWindowDimensions} from 'react-native';
 
-const HOME_TAB_NAME = 'Assistant';
-
-export const useNavigateHome = () => {
+export const useBottomNavigationStyles = (): {height: number} => {
+  const {fontScale} = useWindowDimensions();
```

Comment:
Hey! I didn't know about that! I used `DeviceInfo` which is way more work! Thank you!